apps/docs/content/components/status/Callout.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Callout
+description: A Callout informs users about important changes or persistent conditions. Use this component to communicate to users in a prominent way. Callouts are placed at the top of the page or section they apply to, and below the page or section header.
+category: "status"
+links:
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/Callout/src/Callout.tsx
+---
+
+<Example src="Callout/docs/preview" isOpen />
+
+### Composed Components
+
+A `Callout` uses the following components:
+
+<ComposedComponents components={["Header", "Content", "Button", "RichIcon"]} />
+<br />
+
+A `CompactCallout` uses the following components:
+
+<ComposedComponents components={["Content", "Button"]} />
+
+## Usage
+
+### Variants
+A callout can use different variants.
+
+<Example src="Callout/docs/variant" />
+
+### Hide Icon
+A callout can hide the icon in scenarios where it takes up too much space, such as very small breakpoints.
```

Comment:
> such as very small breakpoints
You are 100% right about this. To make sure people can do this easily, we should make sure that the hideIcon property should be a `ResponsiveProp<boolean>`

this way they can do something like
```
hideIcon={{base: false, md: true}}
```